packages/SketchMorph2-Core.package/M2IconMenuItem.class/class/defaultIconSize.st

```diff
@@ -1,3 +1,3 @@
 as yet unclassified
 defaultIconSize
-	^ Preferences standardMenuFont pointSize * 2
\ No newline at end of file
+	^ 20 asPoint
```

Comment:
Why this change? The old version seems to have a point, having the icon size being dependent on the font size of Squeak.